src/runtime/objmodel.cpp

```diff
@@ -607,8 +624,11 @@ Box* Box::getattr(const std::string& attr, GetattrRewriteArgs* rewrite_args) {
             rewrite_args->out_success = true;
 
         if (rewrite_args) {
-            if (!rewrite_args->obj_hcls_guarded)
+            if (!rewrite_args->obj_hcls_guarded) {
+                if (cls->attrs_offset < 0)
+                    REWRITE_ABORTED("");
```

Comment:
`REWRITE_ABORTED` doesn't actually abort the rewrite, but is just supposed to be a way to figure out where the abort happened; I think you still need to set rewrite_args to NULL.